napari/_tests/test_viewer.py

```diff
@@ -100,8 +100,8 @@ def test_add_layer(make_napari_viewer, layer_class, data, ndim):
     'Vectors': 0,
     'Surface': 0,
     'Tracks': 0,
-    'Points': 9,
-    'Labels': 11,
+    'Points': 8,
+    'Labels': 12,
```

Comment:
here is the decrease of points number of expected methods. I do not know why.

Reply:
A recent PR (https://github.com/napari/napari/pull/4809) added a points-layer short cut and increased the number of methods to 9, from the original 8. It was merged a few days ago, so perhaps this issue is due to a merge conflict?

Reply:
Ooh yeah, I haven't touched the points layer at all. That does look like a problem caused by an improper merge - thanks!